src/backend/joanie/core/models/products.py

```diff
@@ -145,7 +144,8 @@ def get_equivalent_course_run_data(self, visibility=None):
         an equivalent course run with null dates and hidden visibility.
         """
         site = Site.objects.get_current()
-        resource_path = reverse("products-detail", kwargs={"id": self.id})
+        # TODO : Migrate all course run normalization logic to the CourseProductRelation model
+        resource_path = f"/api/v1.0/products/{self.id}/"
```

Comment:
this is weird. Either the url still exists and it should be reversed. Or the url does not exist any more and why do we still use it? Shouldn't we reverse on the course/product relation endpoint url?
I'm reviewing commit by commit :wink: